Ginger/Ginger/Agents/AgentEditPage.xaml.cs

```diff
@@ -166,9 +166,9 @@ private void xTestBtn_Click(object sender, RoutedEventArgs e)
             }
         }
 
-        private void RefreshVirtualAgentCount_Click(object sender, RoutedEventArgs e)
-        {
-            VirtualAgentCount.Content = mAgent.VirtualAgentsStarted().Count;
-        }
+        //private void RefreshVirtualAgentCount_Click(object sender, RoutedEventArgs e)
+        //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6410968)